statsmodels/base/_penalties.py

```diff
@@ -110,13 +110,190 @@ def func(self, params):
         v = np.sqrt(1 + (params / self.dlt)**2)
         v -= 1
         v *= self.dlt**2
-        return np.sum(self.wts * self.alpha * v)
+        return np.sum(self.wts * self.alpha * v, 0)
 
     def grad(self, params):
         v = np.sqrt(1 + (params / self.dlt)**2)
         return params * self.wts * self.alpha / v
 
 
+class SCAD(Penalty):
+    """
+    The SCAD penalty of Fan and Li
+    """
+
+    def __init__(self, tau, c=3.7, wts=None):
+        if wts is None:
+            self.weights = 1.
+        else:
+            self.weights = wts
+        self.tau = tau
+        self.c = c
```

Comment:
For those of us who don't know Fan and Li off the top of our heads, what do c and tau represent?

Reply:
And is there a reason we use `wts` not `weights`? I think `weights` is used more in statsmodels...

Reply:
For now it's inertia, wts is currently used in statsmodels in the super class and I avoided so far making backwards incompatible changes.
naming overall for penalization is a bit inconsistent across fit_regularized and penalized fit.
(I'm avoiding alpha)